src/DssSpell.sol

```diff
@@ -55,104 +45,139 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
+        DssExecLib.setAuctionTimeBeforeReset("WBTC-A"   , 7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WBTC-C"   , 7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WSTETH-A" , 7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WSTETH-B" , 7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("ETH-B"    , 4800 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WBTC-B"   , 4800 seconds);
+        
+        // ilk hole changes (Max concurrent liquidation amount for an ilk)
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-A"    , 40 * MILLION);
+        DssExecLib.setIlkMaxLiquidationAmount("ETH-B"    , 15 * MILLION);
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-A"   , 30 * MILLION);
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-B"   , 10 * MILLION);
+        DssExecLib.setIlkMaxLiquidationAmount("WBTC-C"   , 20 * MILLION);
+        DssExecLib.setIlkMaxLiquidationAmount("LINK-A"   ,  3 * MILLION);
+        DssExecLib.setIlkMaxLiquidationAmount("YFI-A"    ,  1 * MILLION);
+        DssExecLib.setIlkMaxLiquidationAmount("RENBTC-A" ,  2 * MILLION);
+
+        // tip changes (Max keeper incentive in DAI)
+        DssExecLib.setKeeperIncentiveFlatRate("ETH-A"           , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("ETH-B"           , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("ETH-C"           , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("WBTC-A"          , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("WBTC-B"          , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("WBTC-C"          , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("WSTETH-A"        , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("WSTETH-B"        , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("CRVV1ETHSTETH-A" , 250); 
+        DssExecLib.setKeeperIncentiveFlatRate("LINK-A"          , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("MANA-A"          , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("MATIC-A"         , 250);
+        DssExecLib.setKeeperIncentiveFlatRate("RENBTC-A"        , 250); 
+        DssExecLib.setKeeperIncentiveFlatRate("YFI-A"           , 250);
+
+        // dog Hole change (Max concurrent global liquidation value)
+        DssExecLib.setMaxTotalDAILiquidationAmount(70 * MILLION);
 
-        // --- DAI Transfers ---
+        // ---------------------------------------------------------------------
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmbLyNUd#poll-detail
+        // https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-29-september-2022/18143
+        
+        // CRVV1ETHSTETH-A stability fee change (2.0% --> 1.5%) 
+        DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A" , ONE_FIVE_PCT_RATE , true); 
+        
+        // YFI-A DC IAM line change (25M --> 10M)
+        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A" , 10 * MILLION);
```

Comment:
Noting here that dss-auto-line does not automatically adjust this down and it will need to be called permissionlessly by a keeper after the spell is cast to reduce the ceiling.